src/pond/lib/timeseries.js

```diff
@@ -344,12 +344,15 @@ class TimeSeries {
     /**
      * Sets a new underlying collection for this TimeSeries.
      *
-     * @param {Collection}  collection The new collection
+     * @param {Collection}  collection       The new collection
+     * @param {boolean}     isChronological  Causes the chronological
+     *                                       order of the events to
+     *                                       not be checked
      *
-     * @return {TimeSeries}            A new TimeSeries
+     * @return {TimeSeries}                  A new TimeSeries
      */
-    setCollection(collection) {
-        if (!collection.isChronological()) {
+    setCollection(collection, isChronological = false) {
+        if (!isChronological && !collection.isChronological()) {
```

Comment:
I find the naming here a little weird.  if `isChronological` is set to `false`, then we do the check.  That confuses me because I expect `isChronological` to be `true` is we expect this series to be in order. Maybe a better name for this parameter is `checkChronological` or something?

or to have the same sense as it does now, `notChronological` or `disableChronological` or something?